CodeOutputPlugin/Manager/CodeGenerator.cs

```diff
@@ -1006,14 +1006,14 @@ private static void AddFindByNameAssignment(CodeGenerationContext context)
             {
                 context.StringBuilder.AppendLine(
                     $"{context.Tabs}{context.InstanceNameInCode} = this.Visual?.GetGraphicalUiElementByName(\"{context.Instance.Name}\") as " +
-                    $"global::{GetClassNameForType(context.Instance.BaseType, context.VisualApi, context)};");
+                    $"global::MonoGameGum.GueDeriving{GetClassNameForType(context.Instance.BaseType, context.VisualApi, context)};");
```

Comment:
What happens if the instance's base type is a component? Wouldn't that cause this to generate invalid code?